cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RuleBookDetails.scss

```diff
@@ -20,6 +20,7 @@
 $loading_bar_color: white;
 $rule-book-metadata-color: #999999;
 $header-bg-color: #f5f5f5;
+$rule-book-create-fields-required-color: #d40001;
```

Comment:
We already have brand-danger `d15668` is this something that is common for all the `required fields` label? @leacd We should either use the existing red or we should start using this as common color. Right now there are too many reds and maroons that are being used in different places.

Reply:
Talked with @leacd, we want to keep both colors. We can use the existing `brand-danger` for error backgrounds, while this new color will be for error text. I extracted this out to `variables.scss` and called it `cdap-red-text`. Feel free to suggest a better variable name.